src/SFA.DAS.Campaign.Web/Views/Shared/TrainingProvider/Details.cshtml

```diff
@@ -29,6 +29,8 @@
     </div>
     <div class="grid-column-two-thirds">
         <div class="page">
+            <h2 class="@CssViewModel.HeadingLarge @CssViewModel.UtilitiesCss.Margin("Top",0)">Training Information</h2>
```

Comment:
Remove the Cssviewmodel as should use the actual class, can remove the corresponding inject also